tools/scripts/track-pr

```diff
@@ -137,10 +137,15 @@ def cherry_pick_pr(pr_id: str) -> None:
     try:
         # Find and fetch the PR commit and both release branches.
         release_branch = max(
-            line.split()[1]
-            for line in run_capture(
-                "git", "ls-remote", CLONED_REMOTE, "refs/heads/release-*"
-            ).splitlines()
+            (
+                line.split()[1]
+                for line in run_capture(
+                    "git", "ls-remote", CLONED_REMOTE, "refs/heads/release-*"
+                ).splitlines()
+            ),
+            key=lambda branch: [
+                int(part) for part in re.search(r"/release-(\d+)\.(\d+)\.(\d+)$", branch).groups()
```

Comment:
There's a super cool regex om the Semver Page which might also be fun to use. :)

https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string